core/util.rs

```diff
@@ -45,7 +60,7 @@ pub fn parse_schema_rows(
 ) -> Result<()> {
     if let Some(mut rows) = rows {
         rows.set_mv_tx_id(mv_tx_id);
-        let mut automatic_indexes = Vec::new();
+        let mut unparsed_indexes = Vec::with_capacity(10);
```

Comment:
i dont like this allocation but i also didn't want to start changing every schema query to "ORDER BY type" or something so we could get the tables first and indexes last...